infrastructure/main.tf

```diff
@@ -133,32 +116,3 @@ module "data-store-db-v11" {
   common_tags     = "${var.common_tags}"
 }
 
-resource "azurerm_key_vault_secret" "POSTGRES-USER-V11" {
```

Comment:
you probably shouldn't remove these in the same PR as your running application will still be using them and the infra code will go before the config change